src/lib/exports.ts

```diff
@@ -233,15 +352,15 @@ export const exportNoteAsPdfFile = async (
     const printOpts = {
       // Needed for codemirorr themes (backgrounds)
       printBackground: true,
-      // Enable margins if header footer is printed
+      // Enable margins if header and footer is printed!
       // No margins 1, default margins 0, 2 - minimum margins
-      // marginsType: includeFrontMatter ? 0 : 1,
-      pageSize: 'A4', // This could be chosen by user,
+      marginsType: 0, // This could be chosen by user.
+      pageSize: 'A4', // This could be chosen by user.
       // headerFooter: includeFrontMatter ? headerFooter : undefined,
     }
     const pdfBlob = await convertHtmlStringToPdfBlob(htmlString, printOpts)
     const pdfName = `${filenamifyNoteTitle(note.title)}.pdf`
-    downloadBlob(pdfBlob, pdfName)
+    downloadBlob(pdfBlob, pdfName, () => revokeAttachmentsUrls(attachmentUrls))
```

Comment:
downloadBlob executes all statements synchronously so we don't need to pass a callback.

```
downloadBlob(pdfBlob, pdfName)
revokeAttachmentsUrls(attachmentUrls)
```

Even  if we need to pass one, we should make the function to return a promise rather than accepting a callback.

Reply:
True, updated!